kubernetes/launch-job.sh

```diff
@@ -187,8 +209,46 @@ else
   CURRICULUM_CMD="/go_attack/kubernetes/curriculum.sh $RUN_NAME $VOLUME_NAME $CURRICULUM"
 fi
 
+if [ -n "${ONLY_LAUNCH_CURRICULUM:-}" ]; then
+  echo "Launching curriculum only."
+  ctl_job_run --container \
+      "$PYTHON_IMAGE" \
+      $VOLUME_FLAGS \
+      --command "$CURRICULUM_CMD" \
+      --high-priority \
+      --gpu 0 \
+      --name go-curric-"$1" \
+      --replicas 1
+  exit 0
+fi
+
+EXTRA_VICTIMPLAY_GPUS=$((MAX_VICTIMPLAY_GPUS-MIN_VICTIMPLAY_GPUS))
+if [ $EXTRA_VICTIMPLAY_GPUS -gt 0 ]; then
+  EXTRA_VICTIMPLAY_HP_FLAG=""
+  EXTRA_VICTIMPLAY_JOB_NAME="go-train-$1-extra"
+  if [ -n "${EXTRA_VICTIMPLAY_HIGH_PRIORITY:-}" ]; then
+    EXTRA_VICTIMPLAY_HP_FLAG="--high-priority"
+    EXTRA_VICTIMPLAY_JOB_NAME+="-hp"  # So that it doesn't conflict with the non-HP job.
+  fi
+
+  # shellcheck disable=SC2086
+  ctl_job_run --container \
+      "$CPP_IMAGE" \
+      $VOLUME_FLAGS \
+      --command "$VICTIMPLAY_CMD" \
+      --gpu 1 \
+      --name "$EXTRA_VICTIMPLAY_JOB_NAME" \
+      --replicas "${EXTRA_VICTIMPLAY_GPUS}" \
+      $EXTRA_VICTIMPLAY_HP_FLAG
+
+  if [ $MIN_VICTIMPLAY_GPUS -eq 0 ]; then
+    echo "Running extra victimplay jobs only."
+    exit 0
+  fi
```

Comment:
i feel like this is slightly weird because I might expect setting `MIN_VICTIMPLAY_GPUS` to still launch evaluate_loop, train, shuffle_and_export, curriculum, and gating.
not a huge deal, you've kinda documented this behavior already under `-g GPUS, --victimplay-gpus GPUS`

Reply:
Happy to change this is you have a better suggestion for how to only launch extra jobs. I just did this cause it was the easiest thing to do.